pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -27,8 +27,7 @@ type Settings struct {
 }
 
 // FromMetrics converts pmetric.Metrics to Prometheus remote write format.
-func FromMetrics(md pmetric.Metrics, settings Settings) (map[string]*prompb.TimeSeries, error) {
-	c := newPrometheusConverter()
+func (c *PrometheusConverter) FromMetrics(md pmetric.Metrics, settings Settings) (map[string]*prompb.TimeSeries, error) {
```

Comment:
we consider this a breaking API change in for changelog purposes.

Reply:
updated changelog